hugegraph-dist/src/assembly/static/conf/log4j2.xml

```diff
@@ -1,29 +1,65 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<configuration status="error">
+<!-- Config will be auto loaded every 60s -->
+<configuration status="error" monitorInterval="60">
+    <properties>
+        <property name="LOG_PATH">logs</property>
+        <property name="FILE_NAME">hugegraph-server</property>
+    </properties>
+
     <appenders>
         <Console name="console" target="SYSTEM_OUT">
             <ThresholdFilter level="DEBUG" onMatch="ACCEPT" onMismatch="DENY"/>
-            <PatternLayout pattern="%-d{yyyy-MM-dd HH:mm:ss} %-5r [%t] [%-5p] %c %x - %m%n"/>
+            <PatternLayout pattern="%-d{yyyy-MM-dd HH:mm:ss} [%t] [%p] %c{1.} - %m%n"/>
```

Comment:
current log format like this:
![image](https://user-images.githubusercontent.com/17706099/122070325-73198c80-ce28-11eb-9565-edf5f62b246f.png)

removed items:
- cpu time of current thread
- fixed 5 length (useless)
- useless %x (almost)
- full class format --> shortly format

Reply:
LGTM